src/main/kotlin/com/team4099/robot2022/commands/drivetrain/SysIdCommand.kt

```diff
@@ -0,0 +1,163 @@
+package com.team4099.robot2022.commands.drivetrain
+
+import edu.wpi.first.wpilibj.Timer
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+import edu.wpi.first.wpilibj2.command.CommandBase
+import edu.wpi.first.wpilibj2.command.Subsystem
+import java.util.function.BiConsumer
+import java.util.function.Consumer
+import java.util.function.Supplier
+
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+class SysIdCommand : CommandBase {
```

Comment:
this command is not specific to the drivetrain so should move out of the drivetrain folder